hack-sound-server/meson.build

```diff
@@ -0,0 +1,43 @@
+namespace = 'HackSoundServer'
```

Comment:
I figured out that the namespace needs to be `HackSound`, since there is a type named `HackSoundServer` (the GInterface that both the skeleton and the proxy classes implement.) I think this is what caused your problem. That was the reason for the type named `_`. After I made this change, now it is named `HackSound.Server`.

Reply:
I thought that you wanted the namespace to be called exactly HackSoundServer. This namespace is not my favorite name, but I will accept it.

What namespace should be used for the GameStateService? "GameState"?

Reply:
I would also be fine if you renamed the namespace to something else (`HackSoundServer.Server`) using generate_gir, but I think this was what was causing your problem.